elastic/apm-server
# Bump up apm-data to v1.8.1

## Motivation/summary

`host.ip` resource attribute was not processed for OTel events which led to missing of this crucial field in the UI.

## Checklist

~~- [ ] Documentation has been updated~~

## How to test these changes

1. Create a setup with opentelemetry-collector. Use `otlphttp` exporter to send data to APM-Server. For receiver, you can use hostmetricsreciever to collect host stats.
2. Notice that docs indexed in Elasticsearch have the field `host.ip` set (note if the field `labels.host_ip` is set then we are not handling it correctly as all unhandled fields are added as a label).

## Related issues

NA
